node_modules/col-core/lib/util.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright 2015 UC Berkeley (UCB) Licensed under the
+ * Educational Community License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License. You may
+ * obtain a copy of the License at
+ *
+ *     http://opensource.org/licenses/ECL-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an "AS IS"
+ * BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+var _ = require('lodash');
+
+/**
+ * Generate a random string
+ *
+ * @param  {Number}     [letters]   The number of letters that should be in the string, defaults to `10`
+ * @return {String}                 A random string of letters
+ */
+var generateString = module.exports.generateString = function(letters) {
```

Comment:
We might as well use something like https://github.com/klughammer/node-randomstring for this